### DISCUSSION TITLE
如何发表高水平学术论文 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by yufree** [yufree/yufree.cn#418] [General]
如何发表高水平学术论文 - Miao Yu | 于淼 
 https://yufree.cn/cn/2023/04/01/april-fool/

**Comment by yufree** [comment]
`Originally posted by Hs lin on 2023-04-03 08:03:13
        `
[https://uploads.disquscdn.c\...](https://uploads.disquscdn.com/images/f622dbc41995e4c7d6e2b507fe8eb1ee9b52160aead9837fcc9e70800f104107.png)
看起来bing chat不太行

**Reply by yufree** [reply]
`Originally posted on 2023-04-03 12:41:30
        `
哈哈，属实污染了AI数据库了

**Comment by sasazhao** [comment]
ha??? 捂脸

**Comment by wuliring** [comment]
??好像被耍了。。（无奈.jpg）
